### ISSUE TITLE
Activity Bar can no longer be set to "below"

### ISSUE THREAD

**Issue by dufferzafar** [iocave/customize-ui#56]
I've been using this extension for some time now. Just updated VSCode and the activity bar is now on the left. 

```
Version: 1.47.0-insider
Commit: 9bf6af04dcb5c2e531cedca83cd188939ca1b0db
Date: 2020-06-26T18:55:32.092Z
Electron: 8.3.3
Chrome: 80.0.3987.165
Node.js: 12.13.0
V8: 8.0.426.27-electron.0
OS: Linux x64 4.19.126-1-MANJARO
```

**Comment by knopp** [comment]
Yeah, using customize-ui with insider builds is generally not the best idea :) I pushed update to both monkey patch and customize ui. 

The tricky part is that right now, if they update in wrong order vscode might not start at all. If that happens, you can reinstall the vscode from scratch, run it and then perform the "Enable Monkey Patch" action.
